tema4.js

```diff
@@ -0,0 +1,17 @@
+//  const garden = ["gradina", "copac", "floare"]
+
+function getVowelsCount(sentence) {
+    
+    const vowels = ['a', 'e', 'i', 'o', 'u']
+    const result = sentence.split('').reduce((prev, current) => {
+        if (vowels.includes(current.toLowerCase())) {
+            return prev + 1
+        } else {
+            return prev
+        }
+    }, 0)
+    
+ return result
+}
+console.log(getVowelsCount("gradina", "copac", "floare"))
```

Comment:
Aici se presupune sa avem simplu un singur string, nu o lista.
```suggestion
console.log(getVowelsCount("gradina"))
```
Daca vrem sa afisam vocalele la mai multe string-uri atunci facem astfel:
console.log(getVowelsCount("gradina"));
console.log(getVowelsCount("copac"));
console.log(getVowelsCount("floare"))l